.github/workflows/samples-tools/cmd/run-all/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"samples-tools/pkg/utils"
+	"strings"
+)
+
+func main() {
```

Comment:
Same as previous: provide some comments that explain the flags that are expected. In particular, I was confused by what the 0th parameter was supposed to be, until I looked at the experimental.yaml.